<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConfig</class>
 <widget class="QDialog" name="ServerConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qcanserver.qrc">
    <normaloff>:/images/network-icon.png</normaloff>:/images/network-icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pclBtnCloseM">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="pclTabConfigM">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pclTabConfigNetworkM">
    <attribute name="title">
     <string>Network configuration</string>
    </attribute>
    <widget class="QToolBox" name="pclTbxNetworkM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>361</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget1_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>145</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 1</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget2_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>145</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 2</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget3_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 3</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget4_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 4</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget5_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 5</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget6_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 6</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget7_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 7</string>
      </attribute>
     </widget>
     <widget class="QCanInterfaceWidget" name="pclTbxQCanInterfaceWidget8_M">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>101</width>
        <height>145</height>
       </rect>
      </property>
      <attribute name="label">
       <string>CAN 8</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pclGrpNetworkConfM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>451</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>CAN network configuration</string>
     </property>
     <widget class="QComboBox" name="pclCbbNetBitrateM">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>35</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>10 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>100 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>125 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>250 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>500 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>800 kBit/s</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 MBit/s</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="pclLblNetBitrateM">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bit-rate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetEnableM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable network</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetErrorFramesM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Error frames</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetCanFdM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CAN-FD support</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pclCkbNetListenOnlyM">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Listen-only</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pclGrpNetworkStatM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>451</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>CAN network statistics</string>
     </property>
     <widget class="QProgressBar" name="pclPgbStatLoadM">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatMsgM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Messages / sec</string>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatErrM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Error frames:</string>
      </property>
     </widget>
     <widget class="QLabel" name="pclLblStatCanM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CAN frames:</string>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>20</width>
       <height>341</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pclTabConfigServerM">
    <attribute name="title">
     <string>Server configuration</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CAN server port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of CAN networks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>48</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pclTabConfigInfoM">
    <attribute name="title">
     <string>Information</string>
    </attribute>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCanInterfaceWidget</class>
   <extends>QWidget</extends>
   <header>qcan_interface_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qcanserver.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pclBtnCloseM</sender>
   <signal>clicked()</signal>
   <receiver>ServerConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
